Listenbrainz/Model/Feed/FeedMbidMapping.swift

```diff
@@ -10,9 +10,9 @@ import Foundation
 
 // MARK: - FeedMbidMapping
 struct FeedMbidMapping: Codable {
-    let artistMbids: [String]
-    let artists: [FeedArtist]
-    let recordingMbid: String
+    let artistMbids: [String]?
```

Comment:
Not that I have checked, but are you sure this is necessary? You will have to force unwrap and check unnecessarily everywhere otherwise

Reply:
Agreed , commented out artistMbids @akshaaatt

Reply:
@gauravbhardwaj7 sorry if my comment was unclear. I meant that should we make fields nullable? Because in this PR you have made almost everything nullable, which is not good.

Reply:
@akshaaatt I've been checking out response for my feed, found out these parameters to be null
<img width="586" alt="Screenshot 2024-01-01 at 10 29 38 PM" src="https://github.com/metabrainz/listenbrainz-ios/assets/106892217/0a9a6dfe-d510-458f-ad46-e8f431be6d2c">

Reply:
Interesting @gauravbhardwaj7, makes sense then. However, if we are not using any of these fields in the frontend, don't even receive them in the model then.

Reply:
@akshaaatt I think there's something wrong in my feed lol

Reply:
No @gauravbhardwaj7, I don't think so. The empty fields may be because you listen to songs which are a bit unique and are not present in the musicbrainz db. I assume we do not have too many Hindi or Punjabi songs there. It would be best if you stopped thinking and verify stuff now. Do not settle for anything. The api could also be wrong from the backend folks. Question them and get your doubts cleared.

Reply:
yeah @akshaaatt , the purpose of PR mainly was to tackle issues related to listens submitting through soundcloud as few users reported , I guess we have more things to do at the moment.

Reply:
Sure, let's move forward with this PR then and fix these things in the upcoming PRs

Reply:
alright!